docs/LwM2M_Client/Nordic/nRF9160DK.md

```diff
@@ -82,6 +82,7 @@ To connect the board:
 ## Configuring the Client
 
 0. With the board still connected to a serial port interface, open a serial communication program.
+0. Use the `anjay stop` command to stop LwM2M Client and change credentials.
```

Comment:
IMO you can move this point under `Use the anjay command to list possible options:`

Reply:
I guess it won't be comfortable to see command possible options while anjay traces are still incoming

Reply:
So maybe we can add a mention of it in `Use the anjay command to list possible options` point and also in the next point, when we describe how to change credentials?